src/components/Molecules/JoinPoll/ChosenAnimals/ChosenAnimals.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+import {
+  StyleSheet,
+  View,
+  Image,
+} from 'react-native'
+
+import gStyle from '../../../styles'
+import {
+  ColoredSquare,
+} from '../../../../components'
+
+const boxSize = 75
+
+export default (props) => {
+  const styles = StyleSheet.create({
+    parent: {
+      flexDirection: 'row',
+      height: boxSize,
+      width: boxSize * props.animals.length,
+    },
+  })
+
+  return (
+    <View style={styles.parent}>
+      {
+        props.animals.map(animal => (
+          <ColoredSquare key={animal.order} backgroundColor={animal.backgroundColor}>
+            <Image source={{ uri: animal.imageUrl }} />
```

Comment:
I'm not actually seeing any animals show up when I try to run it - are you sure this is working?